openapi/paths/reports@kyc-request-summary.yaml

```
@@ -0,0 +1,54 @@
+servers:
+  - url: 'https://api-sandbox.rebilly.com/experimental/organizations/{organizationId}'
+    description: Sandbox server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+  - url: 'https://api.rebilly.com/experimental/organizations/{organizationId}'
+    description: Live server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+get:
+  x-products:
+    - Reports
+  tags:
+    - Reports
+  summary: Retrieve a KYC requests report
+  operationId: GetKycRequestSummaryReport
+  x-sdk-operation-name: getKycRequestSummary
+  description: |
+    Retrieve a KYC request report by type and by rejection reasons.
+  parameters:
+    - name: periodStart
+      in: query
+      description: Report start date.
+      required: true
+      schema:
+        type: string
+    - name: periodEnd
+      in: query
+      description: Report end date.
+      required: true
+      schema:
+        type: string
+  responses:
+    '200':
+      description: Report was retrieved successfully.
```

nitpick: Avoid past tense
```suggestion
      description: Report successfully retrieved.
```